### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by oddlittlebird**
Added prerequisites and clarified some content. Started a First Steps section, which should be expanded in the future.

Needs more information about what to do once you get it up and working. I added content to index.md, now it is displaying:

+++ url = "/" title = "Syna Theme" date = "2017-09-07" description = "Open Source Theme from Okkur for your next project" +++

Heading 1

----
If you want new Hugo users to be successful, I recommend adding  instructions so they can at least get a "Hello, World!" on the first page. Maybe give them a clue where to find layouts, too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,3 +55,7 @@ images in their own fragment directory, page directory and `static/images`
 directory. Read our [image fallthrough documentation](https://syna.okkur.org/docs/image-fallthrough/) for more info.
 
 Further details read our [full documentation](https://syna.okkur.org/docs).
+
+## First Steps
+
+Open index.md and type. The changes are visible almost immediately at http://localhost:8090/.
```
Comment (mpourismaiel) [inline README.md]: Isn't the default port `1313`?

**Comment by oddlittlebird** [comment]
Yes, that was my mistake. 1313 was blank, and when I started editing the
file in VSC, it opened the file in 8090.

**Comment by mpourismaiel** [comment]
So please change the port and we'll merge it right away.

**Comment by oddlittlebird** [comment]
Done, I think. I made the change and committed it.

**Comment by stp-ip** [comment]
Awesome thanks \o/

**Pull request merged**